.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,13 +2,44 @@ Fixes #
 
 ## Changes
 
-- x
-- y
-- z
+- _Insert list of changes_
 
-## Checks
+## 📸 Looks like
 
-Delete if not applicable:
+_Insert screenshots and/or videos here_
+
+## How to QA
+
+_Update these checklists as necessary_
+
+### 👀 Visual changes
+
+_Insert a detailed list explaining exactly how to QA your changes from a designer’s PoV_
+
+- `git checkout main`
+- `git pull`
+- `git checkout branch-name`
+- `yarn`
+- `yarn storybook`
```

Comment:
```suggestion
- `git fetch`
- `git checkout <pr-branch-name>`
- `yarn`
- `yarn storybook`
```

Why would people want to pull the latest `main` for checking the branch PR? Also, as said in my other comment, checking out the branch if the repo was forked involves more steps. But maybe it's also unlikely that one would have contributers from outside bitcrowd.

For the case where this should be with less friction, one could utilize https://cli.github.com/. From what I know, it's agnostic of the branch origin and can checkout any PR branch with just the PR ID.

Reply:
Aha I see, thanks for the explanation! So far the main complication has been when a branch has been force-pushed due to rebase — without a little technical knowledge that’s a worrying situation where git gives scary messages. I’ll look into the GitHub cli tools, but for now this is already an improvement ✨